### ISSUE TITLE
Resources & MissingKeys handler

### ISSUE THREAD

**Issue by Vincz** [sunfuze/koa-i18next#3]
Hi :)
Would you accept a PR with two additional middlewares?
A resourceHandler & a missing keys handler.
Like this : https://github.com/i18next/i18next-express-middleware/blob/796a85b2feb91f6a7d8ee14c34ae52c107be6efd/src/index.js#L62

but for KOA :)

**Comment by sunfuze** [comment]
of course

**Comment by Vincz** [comment]
Hi! I have the code ready for the two handlers, but I don't know how to export them with the current commonjs module config.
If we switch, it would be a BC and we'll have to do something like this:

    const koaI18next = require('koa-i18next');
    app.use(koaI18next.default(i18next, {...}));

What do you think?

**Comment by sunfuze** [comment]
Hi, you needn't using `.default` to get the middleware. Source code wrote by es6, but it will be converted to commonjs by rollup when publishing. Just like example in README, you can use is as:
```javascript
const koaI18next = require('koa-i18next')
app.use(koaI18next(i18next, options))
```

**Comment by Vincz** [comment]
Ok, but how do I export and use the two new handlers ? 

    export function getMissingKeyHandler() {
        ...
    }

**Comment by sunfuze** [comment]
1. `getMissingKeyHandler` is a function accept config and return a middleware function, example:
```javascript
const koaI18next = require('koa-i18next');
app.use(koaI18next.getMissingKeyHandler(config));
```

2. `getMissingKeyHandler` is a middleware function, example:

```javascript
const koaI18next = require('koa-i18next');
app.use(koaI18next.getMissingKeyHandler);
```

**Comment by Vincz** [comment]
This is actually functions returning middlewares.
The problem is that if we keep the root with the default Koa Middleware, we will not be able to use the two other handlers the way you describe.
If `require('koa-i18next')` returns the koa middleware, we will not be able to use `require('koa-i18next').xxx` for something else. Unless `require('koa-i18next')` returns an object like this:

    {
        default: koaI18nextHandler,
        getMissingKeyHandler: missingKeyHandler,
        getResourcesHandler: resourcesHandler
    }

and it's a BC as we would have to use it like this `require('koa-i18next').default` for the default middleware.

Do you see my problem?

Additionally, when trying to build with the current config, I get:

```
❯ yarn build
yarn run v1.6.0
$ NODE_ENV=production rollup -c

src/index.js → ./lib/index.js...
[!] (babel plugin) Error: It looks like your Babel configuration specifies a module transformer. Please disable it. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
src/index.js
```

I can solve it by setting `modules: false` in `.babelrc`, but if I do so and I export my two handlers like this:

```
export function getResourcesHandler(i18next, options) {...}
export function getMissingKeyHandler(i18next, options) {...}
```

I get the following message during build:

```
❯ yarn build
yarn run v1.6.0
$ NODE_ENV=production rollup -c

src/index.js → ./lib/index.js...
(!) Mixing named and default exports
Consumers of your bundle will have to use bundle['default'] to access the default export, which may not be what you want. Use `output.exports: 'named'` to disable this warning
created ./lib/index.js in 190ms
```

**Comment by sunfuze** [comment]
I updated the exporting code to 
```javascript
export default function koaI18next(i18next, options = {}) {
  return async function i18nextMiddleware(ctx, next) {
  }
}
```

you can export your function as follow
```javascript
koaI18next. getResourcesHandler = function(i18next, options) {...}
koaI18next. getMissingKeyHandler = function(18next, options) {...}
```

**Comment by Vincz** [comment]
PR is here https://github.com/sunfuze/koa-i18next/pull/4

**Comment by sunfuze** [comment]
Resolved #4
